server/src/resolvers/mapping/utils.ts

```diff
@@ -61,8 +64,15 @@ export const buildInputsQuery = (
     return input
   })
 
-export const buildAttributesQuery = (
-  attributes: AttributeWithInputs[],
+export const buildCommentQueryPreV7 = (
+  comment: string,
+): CommentCreateWithoutAttributeInput => ({
+  content: comment,
+  author: { connect: { email: 'admin@arkhn.com' } },
```

Comment:
What if the admin user is not in the DB?
Would it be worth having user optional for a comment and display something like _anonymous_ when the comment has no identified user?

Reply:
This can be done in a later PR if you think it's a good idea but you're sick of working on that!

Reply:
the author of a comment is  not optional so I have to connect the author somehow. I agree that this will break if there is no user "admin@arkhn.com" in the db but it's the easiest way I found to this now. I've discussed it with elsie and nico and we noticed that je issue is bigger than that : when you export a mapping on staging, you want to be able to import it on demo: that will fail if there is no user that matches the author's email. We thought of decorrelating a mapping and its "review" (ie: its comments) but we'll work on that later.